dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jakarta.mail-2.0.feature

```diff
@@ -0,0 +1,9 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jakarta.mail-2.0
```

Comment:
I think this file should be deleted?

Reply:
This feature is included in the mail-2.0 feature. Maybe a rename is needed?

Reply:
https://github.com/OpenLiberty/open-liberty/pull/13751#issuecomment-692003418